packages/fxa-auth-server/grunttasks/ftl.js

```diff
@@ -5,6 +5,17 @@
 'use strict';
 
 module.exports = function (grunt) {
+  // make a copy of the local branding terms available for local development
```

Comment:
Pulled in this grunt task that was recently added to fxa-settings to make sure the latest branding file is available for local dev